server/src/main/java/org/opensearch/indices/replication/common/Timer.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.indices.replication.common;
+
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.io.stream.Writeable;
+import org.opensearch.common.unit.TimeValue;
+
+import java.io.IOException;
+
+/**
+ * A serializable timer that is used to measure the time taken for
+ * file replication operations like recovery.
+ */
+public class Timer implements Writeable {
```

Comment:
"Timer" is a _super_ generic name, used in lots of places (e.g. java.util.Timer, org.opensearch.search.profile.Timer, etc). Is there a more specific name that makes sense now that it is no longer an inner class? Maybe ReplicationTimer?